### PR TITLE
RLS: v0.13.0

### PR DESCRIPTION
**Pull request by cpcloud**

### CODE REVIEW

**Comment by cpcloud** [comment]
Hard work in this release! Thanks @kszucs, @DiegoAlbertoTorres, @jreback, @tsdlovell for your work!

**Comment by kszucs** [comment]
My pleasure!

File: setup.py [hunk 1]
```diff
@@ -27,18 +12,17 @@
...
-    'thrift<=0.9.3',
 ]
 
 sqlite_requires = ['sqlalchemy>=1.0.0,<1.1.15']
 postgres_requires = sqlite_requires + ['psycopg2']
 mysql_requires = sqlite_requires + ['pymysql']
 kerberos_requires = ['requests-kerberos']
 visualization_requires = ['graphviz']
-clickhouse_requires = ['clickhouse-driver>=0.0.8']
+clickhouse_requires = ['clickhouse-driver>=0.0.8', 'clickhouse-sqlalchemy']
```
Comment (kszucs) [inline setup.py]: I think `clickhouse-sqlalchemy` is required only for the `ci` scripts.

**Comment by cpcloud** [comment]
Merging this on green.

**Comment by jreback** [comment]
who hoo!

**Comment by cpcloud** [comment]
Passing appveyor build here: https://ci.appveyor.com/project/cpcloud/ibis-xh5g1/build/1.0.1781

**Pull request closed without merging**